docs/src/main/asciidoc/spring-cloud-stream-support.adoc

```diff
@@ -531,12 +559,139 @@ public Supplier<Message<String>> supply() {
 }
 ----
```

Comment:
If `spring.cloud.stream.default.producer.errorChannelEnabled` is not related to principal method, should we keep same style in line 450 / 486 / 524?

Reply:
Good point, I have update them as the same.